clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/IngestionStats.java

```diff
@@ -328,51 +328,51 @@ public void recordConsumedRecordEndToEndProcessingLatency(double value) {
   }
 
   public double getRecordsConsumed() {
-    return recordsConsumedSensor.getRate();
+    return recordsConsumedSensor.measure(METRIC_CONFIG, System.currentTimeMillis());
   }
 
   public void recordRecordsConsumed() {
     recordsConsumedSensor.record();
   }
 
   public double getBytesConsumed() {
-    return bytesConsumedSensor.getRate();
+    return bytesConsumedSensor.measure(METRIC_CONFIG, System.currentTimeMillis());
   }
 
   public void recordBytesConsumed(double value) {
-    bytesConsumedSensor.record(value);
+    bytesConsumedSensor.record((long) value);
```

Comment:
Nitpick... should we change the function param's type to long rather than casting here? What we're passing is going to be an int or a long, so why potentially lose precision in a conversion to double only to then reconvert it again to long? If you agree, then let's do the same for the other similar cases in this class.

Reply:
Sure, its pointless for this to be double here to do back and forth conversions,